jiraannouncer/views/fcms.py

```diff
@@ -0,0 +1,43 @@
+import hmac
+
+from pyramid.view import view_config
+from sys import hexversion
+
+from ..utils import send, devsay
+import logging
+
+log = logging.getLogger(__name__)
+
+
+@view_config(route_name='fcms', renderer="json")
+def fcms(request):
+    try:
+        data = request.json_body
+    except:
+        log.error("Failed to decode JSON from body. Dump:")
+        log.debug(request.body)
+        devsay("A FCMS payload couldn't be decoded. NoLifeKing, check the logfile!",
+               request)
+        return
+
+    carrier = data['carrier_callsign']
+    target_system = data['calendar_arrivalSystem']
+    departure_system = data['current_starsystem']
+    jump_time = data['calendar_end'] if 'calendar_end' in data else None
+
+    if jump_time is None:
+        send(
+            "#announcerdev",
+            f"[FCMS] Carrier {carrier} cancelled their scheduled jump",
```

Comment:
Should probably include the hexdecoded actual carrier name, in addition to the callsign.

Reply:
Is that the vanityname? Because since I didn't see anything in clear text, I didn't use that.